torch_xla/csrc/tensor_impl.cpp

```diff
@@ -110,7 +114,11 @@ void XLATensorImpl::shallow_copy_from(
 }
 
 at::IntArrayRef XLATensorImpl::sizes_custom() const {
-  const_cast<XLATensorImpl*>(this)->SetupSizeProperties();
+  if (true) { /* TODO(@miladm): replace this with a flag */
+    const_cast<XLATensorImpl*>(this)->SetupSymSizeProperties();
```

Comment:
@Krovatkin Is the plan to always use symsize even for the static ints?

Reply:
No, in his later PR, @miladm is using `is_dynamic` to decide whether we need to create symint nodes or static ints : 

https://github.com/pytorch/xla/pull/3909/files#diff-c4e1dd39b63d78af7c207b2d48ac29553d74214b1c185ae34e084dd2f583879eR197

Reply:
In this PR, @miladm is forcing both sym_sizes_ and sizes_and_strides_ to be filled with upper bounds. There are no real symint nodes yet.